.github/workflows/build.yaml

```diff
@@ -21,6 +21,21 @@ jobs:
         shell: bash
         run: yarn install --immutable
 
+      - name: Create database folder
+        run: mkdir -p .test/db
+
+      - name: Give permissions to test database
+        run: sudo chown -R $USER .test/db
+
+      - name: Init test database
+        run: docker compose up -d
+
+      - name: Wait for database
+        run: docker compose exec db bash -c 'while ! pg_isready -h localhost -p 5432 -U postgres; do sleep 1; done;'
```

Comment:
If it works, it seems okay to merge 👍🏻 As a note, I remember there is a *service container* feature in the GitHub Actions service. It would be nice to see if it's a good idea to utilize this feature in the future when you have some free time.

https://docs.github.com/en/actions/using-containerized-services/about-service-containers